float64[7] ang1
float64[7] ang2
float64[7] ang3
float64[7] ang4
float64[7] ang5
float64[7] ang6
float64[7] ang7
float64[7] ang8
